package.json

```diff
@@ -4,16 +4,17 @@
   "description": "eCommerce Backend portion",
   "main": "index.js",
   "scripts": {
-    "start": "nodemon -r dotenv/config server.js",
+    "start": "NODE_ENV=development nodemon -r dotenv/config server.js",
     "create-db": "node db/setupDatabase.js",
     "seed-db": "node db/seedDatabase.js",
-    "test": "jest --forceExit --runInBand"
+    "test": "NODE_ENV=test jest --forceExit --runInBand"
```

Comment:
These explicit `NODE_ENV` declarations will help CORs know how to handle external requests.